symbolic-examples/stats/SHA256.32

```diff
@@ -1,4 +1,4 @@
 SHA256.32
-Number of constraints: 50566
-Number of variables: 60683
-Number of range lookups: 10996
\ No newline at end of file
+Number of constraints: 51184
```

Comment:
I feel like we forgot to run benchmarks on some previous commit which caused the regression, can you check the result of this benchmark on `main` branch?

Reply:
The benchmark result on the `main` branch is indeed smaller. Somehow, transformations increase the number of range constraints.